transhumandesign/kag-base
# [fixed] Incorrect saw sprite on player join

## Status

- **READY**

## Description

Resolves #1378 & Closes #1395 

The difference between this PR and #1395 is that the saws will also synchronize the saw blade's bloody state. Sync is also done just once instead of every tick.

## Steps to Test or Reproduce
1) Go onto a dedicated server.
2) Create a saw, jump in it to make it bloody, then turn it off.
3) Disconnect and then rejoin the server. With the applied fix, the saw should look exactly the same as before.

Note for reviewers:
Commit 1 is the necessary changes, Commit 2 has no functional change.
